### PR TITLE
Fleshed out course intro

### PR DESCRIPTION
**Pull request by vwilburn**
Added formal objectives, prereqs, and general description.

### CODE REVIEW

File: workshop/README.md [hunk 1]
```diff
@@ -1,9 +1,31 @@
-# Istio 101 workshop
+# Advanced Istio course (name TBD)
```
Comment (linsun) [inline workshop/README.md]: Again, this course is advanced compared with docker and k8s, but this is not an advanced istio course.  This is a istio 101 hands on course.

File: workshop/README.md [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+[Istio](https://www.ibm.com/cloud/info/istio) is an open platform to connect, secure, and manage a network of microservices, also known as a service mesh, on cloud platforms such as Kubernetes in IBM Cloud Container Service. With Istio, manage network traffic, load balance across microservices, enforce access policies, verify service identity on the service mesh, and more.
 
-## Workshop Setup
-- [Exercise 1 - Accessing a Kubernetes cluster with IBM Cloud Container Service](exercise-1/README.md)
+In this course, you can see how to install Istio alongside microservices for a simple mock app called Guestbook. The microservices include a product web page, details, reviews, and ratings. When you deploy Guestbook's microservices into an IBM Cloud Container Service cluster where Istio is installed, you inject the Istio Envoy sidecar proxies in the pods of each microservice.
```
Comment (linsun) [inline workshop/README.md]: > The microservices include a product web page, details, reviews, and ratings.

This is not accurate.  The guest book doesn't contain details/reviews/rating or product page.

File: workshop/README.md [hunk 3]
```diff
@@ -1,9 +1,31 @@
...
+- Download and install Istio in your cluster
+- Deploy the Guestbook sample app
+- Set up the Istio Ingress controller
+- Use metrics, logging and tracing to observe services
+- Perform simple traffic management, such as A/B tests and canary deployments
+- Secure your service mesh
+- Enforce policies for your microservices
+
+## Prerequisites
+You must create a [standard cluster in a Pay-As-You-Go and Subscription account](https://console.bluemix.net/registration/) to complete all the modules in this course. A free cluster in a Trial account does not have access to all the advanced features of this course.
```
Comment (linsun) [inline workshop/README.md]: I don't think this is entirely necessary as a pre-requisites.  User can do every thing in the trial account, the only thing is they need to access the istio-ingress via node-port and not do the front door portion.  cc @szihai

File: workshop/README.md [hunk 4]
```diff
@@ -1,9 +1,31 @@
...
+
+## Prerequisites
+You must create a [standard cluster in a Pay-As-You-Go and Subscription account](https://console.bluemix.net/registration/) to complete all the modules in this course. A free cluster in a Trial account does not have access to all the advanced features of this course. 
 
-If you are not familar with Kubernetes, we recommend you to follow [Kubernetes 101](https://github.com/IBM/kube101) learn Kubernetes.  If you are already familar with Kubernetes, feel free to proceed to exercises below to learn Istio.
+You should have a basic understanding of containers, IBM Cloud Container Service, and Istio. If you have no experience with those, take the following courses:
+1. [Docker essentials: Extend your apps with containers](https://developer.ibm.com/courses/all/docker-essentials-extend-your-apps-with-containers/) 
+2. [Get started with Kubernetes and IBM Cloud Container Service](https://developer.ibm.com/courses/all/get-started-kubernetes-ibm-cloud-container-service/)
+3. [Get started with Istio and IBM Cloud Container Service](https://developer.ibm.com/courses/all/get-started-istio-ibm-cloud-container-service/)
+
```
Comment (linsun) [inline workshop/README.md]: good links, thanks!

File: workshop/README.md (hunk 1 above)
Comment (vwilburn) [reply workshop/README.md]: This is just a placeholder until the formal name is approved.

File: workshop/README.md (hunk 2 above)
Comment (vwilburn) [reply workshop/README.md]: I'll update.

File: workshop/README.md (hunk 3 above)
Comment (vwilburn) [reply workshop/README.md]: When I saw "Ingress," I assumed Kube Ingress. OK, I'll update this.

**Pull request merged**